controller/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN curl -sSL -o /usr/local/bin/etcdctl https://s3-us-west-2.amazonaws.com/opdem
     && chmod +x /usr/local/bin/etcdctl
 
 # install confd
-RUN curl -sSL -o /usr/local/bin/confd https://github.com/kelseyhightower/confd/releases/download/v0.8.0/confd-0.8.0-linux-amd64 \
+ENV CONFD_VERSION 0.9.0
```

Comment:
An `ENV` statement is another Docker image layer, with all the TCP latency that implies. And we don't need to know `CONFD_VERSION` elsewhere in the container environment. Could we make these one-liners?
```sh
RUN CONFD_VERSION=0.9.0 curl -sSL -o /usr/local/bin/confd https://github.com/kelseyhightower/confd/releases/download/v$CONFD_VERSION/confd-$CONFD_VERSION-linux-amd64 \
     && chmod +x /usr/local/bin/confd
```